index.js

```diff
@@ -12,6 +12,10 @@ const main = async () => {
         // Grab the slug if it's a url
         slug = slug.substring(slug.lastIndexOf('/') + 1);
 
+        //project-name for the user agent
+        let user_agent = actions.getInput("project-name")
+        user_agent = user_agent == "REPLACEME" ? slug : `${user_agent} (${slug})`
```

Comment:
Very nitpicky, but can you use single quotes (`'`) instead of the double quotes (`"`)